src/test/converter/comment/specs.json

```diff
@@ -62,6 +62,78 @@
                 "type": "intrinsic",
                 "name": "string"
               }
+            },
+            {
+              "id": 22,
+              "name": "mixedHidden",
+              "kind": 2048,
+              "kindString": "Method",
+              "flags": {
+                "isExported": true
+              },
+              "signatures": [
+                {
+                  "id": 25,
+                  "name": "mixedHidden",
+                  "kind": 4096,
+                  "kindString": "Call signature",
+                  "flags": {
+                    "isExported": true
+                  },
+                  "comment": {
+                    "shortText": "Mixed hidden 2"
+                  },
+                  "parameters": [
+                    {
+                      "id": 26,
+                      "name": "arg1",
+                      "kind": 32768,
+                      "kindString": "Parameter",
+                      "flags": {
+                        "isExported": true
+                      },
+                      "type": {
+                        "type": "intrinsic",
+                        "name": "any"
+                      }
+                    },
+                    {
+                      "id": 27,
+                      "name": "arg2",
+                      "kind": 32768,
+                      "kindString": "Parameter",
+                      "flags": {
+                        "isExported": true
+                      },
+                      "type": {
+                        "type": "intrinsic",
+                        "name": "any"
+                      }
+                    }
+                  ],
+                  "type": {
+                    "type": "intrinsic",
+                    "name": "any"
+                  }
+                }
+              ],
+              "sources": [
+                {
+                  "fileName": "comment.ts",
+                  "line": 70,
+                  "character": 13
+                },
```

Comment:
I couldn't find a way to remove sources from signatures that have been removed.

Reply:
I think this is OK, but will see if I notice an easy way to do it